src/OpenTelemetry.Instrumentation.GrpcNetClient/GrpcTagHelper.cs

```diff
@@ -36,17 +36,11 @@ public static string GetGrpcMethodFromActivity(Activity activity)
             return activity.GetTagValue(GrpcMethodTagName) as string;
         }
 
-        public static Status GetGrpcStatusCodeFromActivity(Activity activity)
+        public static int GetGrpcStatusCodeFromActivity(Activity activity)
         {
-            var status = Status.Unset;
-
             var grpcStatusCodeTag = activity.GetTagValue(GrpcStatusCodeTagName);
-            if (int.TryParse(grpcStatusCodeTag as string, out var statusCode))
-            {
-                status = ResolveSpanStatusForGrpcStatusCode(statusCode);
-            }
-
-            return status;
+            int.TryParse(grpcStatusCodeTag as string, out var statusCode);
+            return statusCode;
```

Comment:
we need some validation that activity has this tag. If it doesn't, or if its not int, then this would return 0 now, which maynot be what we wanted?

Reply:
if the tag doesn't exist, it would return 0. Which would lead to Unset status. I could return -1 if tryparse is valid, and then, in that case we wouldn't set a rpc.grpc.status_code. What do you think?

Reply:
yea. we don't want to accidently set Unset (just because 0 maps to it).
If we cannot find the statuscode, yes, then lets not set it to anything at all.

Reply:
just updated! can you validate?